ember_debug/port.js

```diff
@@ -13,8 +13,20 @@ export default EmberObject.extend(Ember.Evented, {
    * @property uniqueId
    * @type {String}
    */
-  uniqueId: computed('namespace._application.name', function() {
-    return this.get('namespace._application.name');
+  uniqueId: computed('namespace._application.{name,modulePrefix}', 'namespace.applicationId', function() {
```

Comment:
As we discussed briefly in Discord, I think this should be an `applicationName` computed, and we should revert to the old `uniqueId` generation. We then need to pass both the name and id around and look up the apps by the id somehow, rather than name, I think.

Reply:
Sounds good to me @rwwagner90, I'll leave that to you (happy to review and otherwise help out as needed though)...

Reply:
> As we discussed briefly in Discord, I think this should be an `applicationName` computed, and we should revert to the old `uniqueId` generation. We then need to pass both the name and id around and look up the apps by the id somehow, rather than name, I think.

This makes sense to me as well. I was going to leave a comment to this effect after I thought about the idea of doing something like ``app${randomNum}`` last night, but then realized that **probably** wouldn't work for app lookups anymore when switching between apps.

Reply:
@alexhancock I started trying to do this, but I've been so far unsuccessful. Maybe you could take a look and help get this over the finish line? https://github.com/emberjs/ember-inspector/pull/930

Reply:
@alexhancock are you on Discord? It would be good to sync up there.

Reply:
@rwwagner90 I am not on discord, but I have an old account. Username: alexhancock.

I have some Q's for you about the changes in https://github.com/emberjs/ember-inspector/pull/930 

How do I connect with you all on Discord?

Reply:
@alexhancock the Ember community has a discord. I believe you can join via https://discordapp.com/invite/zT3asNS

Reply:
Cool! I see it now, but I don't have permissions to send messages in the `dev-ember-inspector` channel

![screen shot 2019-01-29 at 11 42 48 am](https://user-images.githubusercontent.com/427516/51935589-1504bf80-23bb-11e9-91eb-402fb8e602c7.png)

Reply:
Heading into a meeting for an hour, but I really only have one question about the changes in #930 so I will just post it here! :P 

Why all the changes to the logic in `app/port.js` and why remove the `sendAppList` logic? 

As far as I can see that code being altered/removed is connected to why populating the list of apps and switching apps no longer works in that changeset. Are the changes born from a desire to clean things up or implement the original app list population logic in a different way?

Happy to help on a revised version of this change after my lunch meeting.

Reply:
@alexhancock the app list was using the `uniqueId` and the names as the same thing. We need a reworking of it to support sending both `applicationId` and `applicationName`, like the PR is doing. We already have a `for` loop that loops through the applications, so if we can find a way to use that to populate the apps, that would be ideal. Ping me on discord I am `rwwagner90` there as well. I see two `alexhancock` so I am not sure which is you. You have to go through the `#welcome` and `#setup-profile` channels to get permissions to everything.

Reply:
Also, another option would be to revert my original change, release a new version and then re-introduce the changes once we get the issues resolved.

Not sure how close we feel we are on fixing the issues - I'll leave it up to you.

Reply:
@alexhancock let's discuss via discord, it will make things quicker